src/I2cAnalyzer.cpp

```diff
@@ -78,20 +78,32 @@ void I2cAnalyzer::GetByte()
     frame.mStartingSampleInclusive = starting_sample;
     frame.mEndingSampleInclusive = result ? potential_ending_sample : last_valid_sample;
     frame.mData1 = U8( value );
+    bool did_ack{ false };
 
     if( !result )
+    {
+        framev2.AddString( "error", "missing ack/nak" );
         frame.mFlags = I2C_MISSING_FLAG_ACK;
+    }
     else if( ack_bit_state == BIT_HIGH )
+    {
         frame.mFlags = DISPLAY_AS_WARNING_FLAG;
```

Comment:
let's remove this line. I double checked the Logic codebase, `DISPLAY_AS_WARNING_FLAG` was never used.